README.md

```diff
@@ -97,6 +95,21 @@ The Checkout application supports [bundles](https://docs.commercelayer.io/develo
 
 If the order has the attribute `cart_url` set, a "Return to cart" link will be displayed.
 
+### Order refresh
+
+When the checkout is opening, after getting the organization settings, we are going to refresh the order if some conditions are met. Below a flowchart shows how the logic works:
```

Comment:
When the checkout is opening, after getting the organization settings, the order might be refreshed if some conditions are met. Check the flowchart below to better understand how this logic works: